### PR TITLE
ODPM-32 - use full agency names

### PR DESCRIPTION
**Pull request by trawick**
The full agency names are in a new ``.csv`` file under ``md/data`` which is used during import to update the pandas ``DataFrame`` with the full names.  This ``.csv`` also contains census ``GEOID`` values for some agencies, but those values aren't yet used.

The dataset facts in the MD landing page are updated to reflect the agency names and, as of a prior ticket, the dropping of 2012 data.

Lastly, the Python test coverage requirement is increased by a trivial amount which reflects a marginal coverage improvement with this pull request.

### CODE REVIEW

File: md/templates/md.html [hunk 1]
```diff
@@ -47,24 +47,24 @@
       <th class="text-right">Stops</th>
     </tr>
     <tr>
-      <td><a href="{% url "md:agency-detail" "80" %}">MSP</a></td>
-      <td class="text-right">641,952</td>
+      <td><a href="{% url "md:agency-detail" "77" %}">Maryland State Police</a></td>
```
Comment (copelco) [inline md/templates/md.html]: Do we know why the IDs changed for the Maryland agencies?

Comment (trawick) [inline md/templates/md.html]: Look at the first two changes:  80/MSP  and 77/Montgomery turned into 77/Maryland State Police and 80/Montgomery County Police Department.  MSP came after Montgomery alphabetically but Maryland comes before.  The table is ``ORDER BY agency_description``.

File: md/data/MD_agencies.csv [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+TANEYTOWN,Taneytown Police Department,16000US2476725
+THURMONT,Thurmont Police Department,16000US2477825
+TOWSONU,Towson University Police Department,
+TRAPPE,Trappe Police Department,16000US2478575
+UBALTPD,University of Baltimore Police Department,
+UMARLBORO,Upper Marlboro Police Department,16000US2479875
+UMB,"University of Maryland, Baltimore Police Department",
+UMBC,"University of Maryland, Baltimore County Police Department",
+UMCP,"University of Maryland, College Park Police Department",
+UMES,University of Maryland Eastern Shore,
```
Comment (rlconley) [inline md/data/MD_agencies.csv]: Is there a reason only these three departments are in quotes?

Comment (trawick) [inline md/data/MD_agencies.csv]: as discussed in hipchat: that is CSV's way to handle commas in the cell

**Comment by rlconley** [comment]
:shipit:

**Pull request merged**